modules/cluster-logging-must-gather-about.adoc

```diff
@@ -11,9 +11,6 @@ For your OpenShift Logging environment, `must-gather` collects the following inf
 
 * project-level resources, including pods, configuration maps, service accounts, roles, role bindings, and events at the project level
 * cluster-level resources, including nodes, roles, and role bindings at the cluster level
-* OpenShift Logging resources in the `openshift-logging` and `openshift-operators-redhat` namespaces, including health status for the log collector, the log store, the curator, and the log visualizer
+* OpenShift Logging resources in the `openshift-logging` and `openshift-operators-redhat` namespaces, including health status for the log collector, the log store, and the log visualizer
```

Comment:
missing period at the end.

Reply:
Thanks, @jc-berger. These are sentence fragments, which don't take periods. However, I've capitalized the first letter of each list item.

IBMSG: `Punctuation in lists
Write lists so that all or none of the items start with complete sentences. If list items comprise only complete
sentences, include a period after each sentence. Except in definition lists, do not include any end punctuation
if list items comprise only sentence fragments.`
IBMSG: `Capitalization in lists
Capitalize the first word of each item in a vertical list.`